docs/source/attributes.rst

```diff
@@ -464,6 +464,10 @@ nginx
 
   -  **Default**: ``10``
 
+-  |app['webserver']['enable_upgrade_method']|_
+
+  -  **Default**: ``false``
+
```

Comment:
You're trying to create a link here, which doesn't refer to anything in footnotes. Since this is a custom parameter, I assume it's not the case. So can you please document it properly? Something like:

```
-  ``app['webserver']['enable_upgrade_method']``

  -  **Supported values:** ``true``, ``false``
  -  **Default:** ``false``
  -  [Some description]
```